openmeter/billing/worker/subscription/sync.go

```diff
@@ -629,6 +629,15 @@ func (h *Handler) getPatchesForExistingLineOrHierarchy(existingLine billing.Line
 }
 
 func (h *Handler) getPatchesForExistingLine(existingLine *billing.Line, expectedLine *billing.Line, invoiceByID InvoiceByID) ([]linePatch, error) {
+	// Lines can be manually marked as ignored in syncing, which is used for cases where we're doing backwards incompatible changes
+	if ignore, ok := expectedLine.Annotations[billing.AnnotationSubscriptionSyncIgnore]; ok && ignore == true {
+		return nil, nil
+	}
```

Comment:
We should be careful using this, as the line in question might have not yet been synchronized to billing, thus those lines will never get synced.